distributed/src/main/scala/asobu/distributed/service/Controller.scala

```diff
@@ -16,7 +17,9 @@ trait Controller {
    *
    * @return
    */
-  def name: String = getClass.getSimpleName.stripSuffix("$")
+  def name: String = getClass.getName.split('$').last.split('.').last //todo: find a better way to get the canonical name of a class (getClass.getCanonicalName doesn't work for classes inside an object
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Usage of last on collections.](https://www.codacy.com/app/iHeartRadio/asobu/file/2373689176/issues/source?bid=3037259&fileBranchId=3224055#l20)